keras/testing/test_utils.py

```diff
@@ -4,29 +4,78 @@
 def get_test_data(
     train_samples, test_samples, input_shape, num_classes, random_seed=None
 ):
-    """Generates test data to train a model on.
+    """Generates synthetic test data for training a model,
```

Comment:
The header line should fit on one line -- you can add a more detailed description as a paragraph below